docs/SEND_MESSAGE_USING_ETHEREUM_PUBLIC_KEY.md

```diff
@@ -0,0 +1,73 @@
+# Send message only knowing recipient's Ethereum Public Key
+
+## Variables
+
+- `A` is Alice's Ethereum root HD public key
+- `B` is Bob's Ethereum root HD public key
+- `a` is the private key of `A`, and is only known by Alice,
+- `b` is the private key of `B`, and is only known by Bob.
+
+## Properties
+
+a. Alice knows Bob's Ethereum root HD public key `B`
+b. Alice wants to send message `M` to Bob using waku,
+c. Bob can find `M` from waku store,
+d. Bob can recognize he is `M`'s recipient when relaying it via waku relay,
+e. Carole cannot read `M`'s content even if she is storing it or relaying it.
+
+### Out of scope
+
+The solution **may not** not have the following properties:
+
+i. Carole must not know Bob is the recipient of a message
+ii. Bob must not be able to deduce Alice's Ethereum root HD public key `A`
```

Comment:
Is this definitely the property we want? I can see the rationale, but also seems asymmetrical. So A knows B but B doesn't know A?

Reply:
It's part of the out-of-scope section, meaning that we do not want to enforce this rule.
It may be clearer if I just remove it. Will do.